examples/FirebasePush_ESP8266/FirebasePush_ESP8266.ino

```diff
@@ -37,28 +34,18 @@ void setup() {
   Serial.println();
   Serial.print("connected: ");
   Serial.println(WiFi.localIP());
-
-   // add a new entry.
-  FirebasePush push = fbase.push("/logs", "{\".sv\": \"timestamp\"}");
-  if (push.error()) {
-      Serial.println("Firebase push failed");
-      Serial.println(push.error().message());  
-      return;
-  }
-
-  // print key.
-  Serial.println("Name: " + push.name());
-
-  // get all entries.
-  FirebaseGet get = fbase.get("/logs");
-  if (get.error()) {
-      Serial.println("Firebase get failed");
-      Serial.println(push.error().message());  
+  
+  Firebase.begin("example.firebaseio.com", "auth_or_token");
+
+  // add a new entry.
+  String name = Firebase.push("/logs", "{\".sv\": \"timestamp\"}");
+  if (Firebase.failed()) {
+      Serial.println(" push failed");
+      Serial.println(Firebase.error());  
       return;
   }
-  // Print written timestamp.
-  String data = get.json()[push.name()];
-  Serial.println("Timestamp:" + data);
+  Serial.print("pushed: ");
```

Comment:
Yes, I think I will adapt the same to show all the api (but stream). Maybe in a different PR?